.github/workflows/package-workflow.yml

```diff
@@ -2,44 +2,36 @@
 on:
   push:
     branches:
-      - master
+    - master
     paths:
-      - "scripts/actions/build-deb/"
-      - ".github/workflows/package-workflow.yml"
+    - scripts/actions/build-deb
+    - .github/workflows/package-workflow.yml
   pull_request:
     paths:
-      - "scripts/actions/build-deb/"
-      - ".github/workflows/package-workflow.yml"
+    - scripts/actions/build-deb
+    - .github/workflows/package-workflow.yml
 
-name: "Release Packages"
+name: 'Release Packages'
 jobs:
   linux-packages:
-    name: "Linux Packages"
+    name: 'Linux Packages'
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@master
+
     - uses: actions/setup-node@master
       with:
         version: 12.x
 
-    - name: 'Install Yarn 1.17.3'
-      run: |
-        curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-        echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-        sudo apt-get update && sudo apt-get install yarn=1.17.3-1 --no-install-recommends
-
-    - name: 'Build'
+    - name: 'Build the standard bundle'
       run: |
         set -ex
         node ./scripts/run-yarn.js build:cli
+
+    - name: 'Build the dist directory'
+      run: |
         ./scripts/build-dist.sh
 
-    - name: 'Build packages'
+    - name: 'Build the deb archive'
       uses: ./scripts/actions/build-deb
-
-    - name: 'Upload packages'
```

Comment:
Why remove this? It makes the artifacts available for download on the build page.

Reply:
Hmm I wasn't sure whether it was useful to store the deb for each commit on master (vs just validating that it can be built), what do you think?